### PR TITLE
Made error toast dynamic on review form page

### PR DESCRIPTION
**Pull request by TheKillerRex27**
you can check this one that was used for the previous cohort

# Fixes Issue/Linear Ticket

> This could be an existing issue or a linear ticket
>
> - Github Issue Example: My PR Closes #{ISSUE}
> - Linear Ticket Example: Fixes ID-#{ISSUE}

# Changes proposed

> Talk about the things you did eg. files changes, dependencies installed e.t.c

# Check List (Check all the applicable boxes)

:rotating_light:Please review the [style guide for contributing](add link here) and [guidelines for contributing](add link here) to this repository.

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.
- [x] I am making a pull request against the **main branch** (left side).
- [x] My commit messages styles matches our requested structure.
- [x] My code additions will fail neither code linting checks nor unit test.
- [x] I am only making changes to files I was requested to.

# Screenshots/ Videos

### CODE REVIEW

File: modules/dashboard/component/reviews/ReviewForm.tsx [hunk 1]
```diff
@@ -11,6 +11,7 @@ import { postReviewByProductId } from '../../../../http/api/controllerReview';
 import { useRouter } from 'next/router';
 import ReviewSentModal from '../../../../components/Modals/ReviewSentModal';
 import { toast } from 'react-toastify';
+import { rating } from '@modules/marketplace/component/filter/data';
```
Comment (chumex412) [inline modules/dashboard/component/reviews/ReviewForm.tsx]: you imported this but it's not being used why?

**Review by chumex412 — COMMENTED** [review]
reasons

File: modules/dashboard/component/reviews/ReviewForm.tsx (hunk 1 above)
Comment (TheKillerRex27) [reply modules/dashboard/component/reviews/ReviewForm.tsx]: Must have been an automatic import my vscode did by mistake
I'll remove it now

Comment (TheKillerRex27) [reply modules/dashboard/component/reviews/ReviewForm.tsx]: Done

**Review by dev-bazz — APPROVED** [review]

**Pull request merged**